tests/phpunit/integration/api/widget/beansSetupWidget.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Tests for beans_setup_widget()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Widget
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Widget;
+
+use Beans\Framework\Tests\Integration\API\Widget\Includes\Beans_Widget_Test_Case;
+
+require_once dirname( __FILE__ ) . '/includes/class-beans-widget-test-case.php';
+
+/**
+ * Class Tests_BeansSetupWidget
+ *
+ * @package Beans\Framework\Tests\Integration\API\Widget
+ * @group   api
+ * @group   api-widget
+ */
+class Tests_BeansSetupWidget extends Beans_Widget_Test_Case {
+
+	/**
+	 * Test beans_setup_widget() should return false when the widget ID can't be found.
+	 */
+	public function test_should_return_false_when_widget_id_not_found() {
+		global $_beans_widget_area;
+
+		beans_register_widget_area( array( 'id' => 'test_sidebar' ) );
+		$this->add_test_widget_to_test_sidebar();
+
+		_beans_setup_widget_area( 'test_sidebar' );
+
+		// Advance the widget counter to a non-existent widget.
```

Comment:
Should we standardize on pointer or counter?

Reply:
Let's go with `pointer`, as it's an integer that locates a particular widget within the array.